apps/src/tutorialExplorer/roboticsButton.jsx

```diff
@@ -34,7 +35,7 @@ const RoboticsButton = React.createClass({
           </a>
         </div>
         <div style={roboticsTextStyle}>
-          Got robots? <a href="/learn/robotics">Use these activities</a> and make a tangible Hour of Code for students of any age!
+          {i18n.roboticsButtonText()}
```

Comment:
I'd make extra sure this works correctly. I would have guess React doesn't let you do this without using `dangerouslySetInnerHtml`